### ISSUE TITLE
Prometheus metrics causes a surge in Sysdig cost. 

### ISSUE THREAD

**Issue by oosawak1975** [ibm-cloud-docs/containers#2583]
If users use the community ingress, Prometheus is enable by default. Unless they explicitly disable Prometheus, nginx will produce Prometheus metrics intensively. It can be found at the metrics scape_*/metricCount.prometheus then segment by proc.name or container.name.

Unfortunately, this info is not documented anywhere in IBM Cloud docs.
https://www.ibm.com/cloud/blog/migration-of-ibm-ingress-resources-to-the-kubernetes-ingress-controller
https://cloud.ibm.com/docs/containers?topic=containers-ingress-types#alb-image-same

After disabling particular metrices it will back to normal billing. 

In short, Prometheus is enable by default in the community ingress. It has to be explicitly disable, if they don't want to get its metrics. 

Please consider to add this information.
